src/main/java/za/co/neslotech/spring/trainspringbootinsight/TrainSpringBootInsightApplication.java

```diff
@@ -0,0 +1,13 @@
+package za.co.neslotech.spring.trainspringbootinsight;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TrainSpringBootInsightApplication {
+
+    public static void main(String[] args) {
```

Comment:
We follow the standard of marking all local variables and method parameters as final 

```suggestion
    public static void main(final String[] args) {
```